templates/YAML/challenge_staging_site.yaml

```diff
@@ -0,0 +1,55 @@
+# These are ACL lists that correlate to actual permissions
+# on synapse
+- view_acl: &view
+    - READ
+
+- download_acl: &download
+    - READ
+    - DOWNLOAD
+
+- edit_acl: &edit
+    - READ
+    - DOWNLOAD
+    - UPDATE
+    - CREATE
+
+- edit_and_delete_acl: &edit_and_delete
+    - READ
+    - DOWNLOAD
+    - UPDATE
+    - CREATE
+    - DELETE
+
+- admin_acl: &admin
+  - READ
+  - DOWNLOAD
+  - UPDATE
+  - CREATE
+  - DELETE
+  - CHANGE_SETTINGS
+  - MODERATE
+  - CHANGE_PERMISSIONS
+
+- &admin_team
+  type: Team
+  id: 3379097
+  name: Sage Bio DREAM Curators
+
+- &service_account
+  type: User
+  id: 3330862
```

Comment:
If we follow the API, shouldn't be called "principalId"?

Reply:
@BrunoGrandePhD  had suggested it [here](https://github.com/Sage-Bionetworks/synapse-formation-templates/pull/5#discussion_r614216374).  Im open for either.

Reply:
I can go either way. I like the consistency of using `name` and `id` for entities and principals, but consistency with the API might be more important. I suppose that we could easily have a mapping between various types (_e.g._ `Team`, `Folder`) and the name of the ID property (_e.g._ `principalId`, `entityId`, respectively). Since consistency with the API was reiterated during the code review last Friday, I'm willing to concede on my earlier comment about using `id` everywhere.